RevenueCatUI/Templates/V2/ViewHelpers/BadgeModifier.swift

```diff
@@ -58,14 +59,17 @@ fileprivate extension View {
                 VStack(alignment: .leading) {
                     VStack {
                         ComponentsView(componentViewModels: badge.badgeViewModels, onDismiss: {})
+                            .padding(badge.stack.value.padding.edgeInsets)
                             .backgroundStyle(badge.backgroundStyle)
                             .shape(border: nil, shape: effectiveShape(badge: badge))
                     }
                     .fixedSize()
                     .padding(effectiveMargin(badge: badge).edgeInsets)
                     .alignmentGuide(
                         effetiveVerticalAlinmentForOverlaidBadge(alignment: badge.alignment.stackAlignment),
-                        computeValue: { dim in dim[VerticalAlignment.center] })
+                        computeValue: { dim in
+                            dim[VerticalAlignment.center] + effetiveYTranslationForOverlaidBadge(badge: badge)
```

Comment:
This is the magic to position the overlaid badge in the middle of the stack's border